### ISSUE TITLE
Scripting behaviour changed?

### ISSUE THREAD

**Issue by MVarland** [skullernet/q2pro#297]
### Important information
I hadn't updated q2pro for a while and noticed a new behaviour for my scripts. Possibly related to the 14th of Feb commit?

### Reproduction steps
Using the latest q2pro I get a new behaviour for **kill[made]** with the below code

```
trigger "set round[on] 0;set kill[#] 0" "*$name$ entered the game*"
trigger "set round[on] 1;set kill[#] 0" "*ACTION!*"
trigger "set round[on] 2" "*round is over*"

alias kill[made] "inc kill[#];if $round[on] == 1 then ${qt}roundkill[msg]${sc}roundkill[radio]${qt}"
alias roundkill[radio] "if $kill[#] <= 10 then radio ${kill[#]} else radio enemyd"
alias roundkill[msg] "say_team ${qt} [ ${kill[#]} ] ${roundkill[say]} [  %K ] ${qt}"
```

### Expected behavior
roundkill[msg] and roundkill[radio] should be called

### Actual behavior
Player says roundkill[msg];roundkill[radio]

**Comment by skullernet** [comment]
Yes, this is due to the fix for issue #290. `if` command now inserts raw command string parsed from its branches, which after initial macro expansion becomes `"roundkill[msg];roundkill[radio]"` (notice double quotes). If you remove `${qt}` macros it should work as before. Perhaps I should also add code to manually strip quotes in cases like this for backward compatibility.

**Comment by ndit-dev** [comment]
A switch, cvar or other method for backwards compatibility would be nice
